scripts/build_natura_raster.py

```diff
@@ -101,7 +100,8 @@ def get_transform_and_shape(bounds, res, out_logging):
         _logger.info("Stage 2/5: Get transform and shape")
     left, bottom = [(b // res) * res for b in bounds[:2]]
     right, top = [(b // res + 1) * res for b in bounds[2:]]
-    shape = int((top - bottom) // res), int((right - left) / res)
+    # x: right-left, y: top-bottom
+    shape = int((right - left) // res), int((top - bottom) // res)
```

Comment:
@ekatef are you sure about switching the order of `x` `y`? Normally in GIS applications, the `y` axis is the first dimension (as in a matrix) and `x` the second which is indeed confusing.